src/components/UI/Display/MAPI/clusters/ClusterDetail/ClusterDetailCounter/__tests__/__snapshots__/index.tsx.snap

```diff
@@ -25,7 +25,7 @@ exports[`ClusterDetailCounter renders a counter with a pluralized label 1`] = `
         </div>
       </div>
       <span
-        class="StyledText-sc-1sadyjn-0 bTNMGz sc-kiwQCO jLIoKV"
```

Comment:
This is not right... I have to update the snapshots even without making changes to the components. This is probably because we no longer have the styled components `babel` plugin (since we switched to `swc`)